packages/jest-haste-map/src/__tests__/index.test.js

```diff
@@ -519,7 +520,7 @@ describe('HasteMap', () => {
       expect(data.map.get('fbjs')).not.toBeDefined();
 
       // cache file + 5 modules - the node_module
-      expect(fs.readFileSync.mock.calls.length).toBe(6);
+      expect(fs.readFileSync.mock.calls.length).toBe(7);
```

Comment:
increased since we use `fs.readfile` to read `package.json` instead if importing it

Reply:
there's a comment above that needs updating as well

Reply:
updated